modules/master/locals.tf

```diff
@@ -21,5 +21,5 @@ locals {
     vnic_attachment.vnic_id
   ]
    
-  formatted_list_of_master_ids = "${formatlist("instance.id='%s'", data.oci_core_instance.master.*.id)}"
+  formatted_list_of_master_ids = formatlist("instance.id='%s'", data.oci_core_instance.master.*.id)
```

Comment:
Have you tested this with multiple masters? We need this for the list of ids so we can use it as the rule for the dynamic group

Reply:
I haven't tested with multiple masters but it should work fine, AFAIR in HCL2 they removed the need for ${} to wrap expressions

Reply:
Unless you need to do string concat and this syntax is still valid and works. Can we please keep this for the time being until we've resolved multi-master?

Reply:
Yeah OK I'll revert it - I think the old syntax is being deprecated but unsure when and it doesn't need to change today